cmd/create/cluster/cmd.go

```diff
@@ -250,6 +255,20 @@ func init() {
 			"Leave empty to use an auto-generated one.",
 	)
 
+	flags.StringVar(
+		&args.oidcEndpointUrl,
+		"oidc-endpoint-url",
+		"",
+		"Endpoint url for BYO OIDC config",
+	)
+
+	flags.StringVar(
+		&args.boundServiceAccountSigningKeySecretArn,
+		"bound-service-account-signing-key-secret-arn",
+		"",
+		"AWS Secrets Manager ARN for retrieval of bound service account signing key",
```

Comment:
Is using "bound service account signing key" exposing too much of the STS implementation detail than necessary? Something like "oidc-private-key" might be sufficient.

cc @jharrington22

Reply:
+1 --oidc-private-key-arn. Probably a good idea to end with the -arn suffix so its clear to the user its not a path or plain text key.

Reply:
At first I thought saying private key was worse haha. Will change this